src/rmq/CMakeLists.txt

```diff
@@ -23,13 +23,17 @@ endif()
 
 target_compile_definitions(rmq PRIVATE USES_LIBRMQ_EXPERIMENTAL_FEATURES)
 
-target_include_directories(rmq PUBLIC rmqt rmqp rmqa)
+target_include_directories(rmq PUBLIC $<INSTALL_INTERFACE:include> $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/rmqt>)
+target_include_directories(rmq PUBLIC $<INSTALL_INTERFACE:include> $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/rmqp>)
+target_include_directories(rmq PUBLIC $<INSTALL_INTERFACE:include> $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/rmqa>)
```

Comment:
These generator expressions are required if we creating a relocatable package (with no absolute paths) which uses `install(TARGETS)` and `install(EXPORT)` later to produce the `rmqcppTargets.cmake` file.